modules/root/protected_root_api.rb

```diff
@@ -0,0 +1,42 @@
+class Proxy::ProtectedRootApi < Sinatra::Base
+  helpers ::Proxy::Helpers
+
+  # authorize_with_trusted_hosts
+  # authorize_with_ssl_client
+
+  get "/v2/features" do
+    begin
+      enabled_plugins = ::Proxy::Plugins.instance.select do |p|
+        p[:name] != :foreman_proxy && \
+          p[:state] == :running && \
```

Comment:
Later I realized that sending the state can be useful. It would mean the server could detect misconfigurations (which is now done in the installer). That does mean the server must always filter on running plugins so it'd be good to get it in on the first go.

Reply:
That makes sense.  I think we'd want to ignore dynamic capabilities if its not running?

Reply:
Good thought